.github/workflows/master.yml

```diff
@@ -1,4 +1,20 @@
-name: Kyuubi
+name: CI
+#
```

Comment:
The workflows yamls has different style of empty line numbers before/after license header, could you please make them consistent?